Wikipedia/assets/index.js

```diff
@@ -1181,38 +1202,40 @@ var newCaption = function newCaption(title, headerText) {
 };
 
 /**
- * @param {!Document} document
+ * @param {!Window} window
  * @param {!Element} content
  * @param {?string} pageTitle
  * @param {?boolean} isMainPage
  * @param {?string} infoboxTitle
  * @param {?string} otherTitle
  * @param {?string} footerTitle
+ * @param {?FooterDivClickCallback} footerDivClickCallback
  * @return {void}
  */
-var collapseTables = function collapseTables(document, content, pageTitle, isMainPage, infoboxTitle, otherTitle, footerTitle, footerDivClickCallback) {
+var collapseTables = function collapseTables(window, content, pageTitle, isMainPage, infoboxTitle, otherTitle, footerTitle, footerDivClickCallback) {
   if (isMainPage) {
     return;
   }
 
   var tables = content.querySelectorAll('table');
-  for (var i = 0; i < tables.length; ++i) {
+
+  var _loop = function _loop(i) {
     var table = tables[i];
 
     if (elementUtilities.findClosestAncestor(table, '.app_table_container') || !shouldTableBeCollapsed(table)) {
-      continue;
+      return 'continue';
```

Comment:
@niedzielski Assuming babel changed this (and below in this method)? Works fine, but do you know what bit got flipped to turn this on?